src/env.c

```diff
@@ -0,0 +1,76 @@
+#include "libft.h"
+#include "env.h"
+
+t_env	env_create(void)
+{
+	t_env	*env;
+
+	env = ft_calloc(sizeof(t_env), 1);
+	if (!env)
+		return (NULL);
+	env->hm_store = get_hm_store();
+	if (!env->hm_store)
+		return (NULL);
```

Comment:
make sure that you destroy the 'env' as well in case the condition on line 12 fails